### ISSUE TITLE
Blur value zero/off

### ISSUE THREAD

**Issue by ASchneiderBR** [Xaymar/obs-StreamFX#625]
### Explain the Feature/Enhancement

Hello.

Please, if possible, allow us to set blur values to zero and use no resources. Now it is only possible to set a minimum of 1, blurring the image a bit.

### What makes this feature/enhancement useful for StreamFX and its users?

When this value is set, it acts the same as setting the filter to hidden, no resources are used at all. This would be great to animate blur in/out effects using Exeldro's move tool, so we can just leave it on and change its blur value to do some really cool effects.

Thank you!

### Checklist

- [ ] I am offering a monetary bounty/reward for the implementation of this Feature/Enhancement.
- [ ] There is no other way to currently do this in StreamFX or OBS Studio.

**Comment by Xaymar** [comment]
Just turn off the filter?

**Comment by tatunts** [comment]
Hello, thanks for the reply.

It is really hard to animate blur in/out smoothly using Exeldro's move tool. The problem is to sequence an animation that has to change your blur filter visibility then apply a value, for example, to blur in smoothly, I'll have to turn the blur filter on, then change its value, the same applies in reverse.

I think the best way would be to have blur filter always on, starting from value zero (no blur applied at all), then just animate its value.

I was able to kind of make this blur in/ou effect to work by adding a blurred out source mirror copy on top then show/hide it using fade transitions, not ideal.

The main goal here is to smoothly animate a source from no blur at all to a specific value (I think only possible using Exeldro's move tools)

Not sure if I was able to explain it, since English is not my mother language, sorry.

Thank you very much!
